### PR TITLE
docs: mention running the CLI after installation

### PR DESCRIPTION
**Pull request by siddiqkaithodu**
Added an additional universal entry point command for spinning up crawlee

### Description

- Added an additional crawlee starting command

### Issues

- Related: https://github.com/apify/crawlee-python/issues/285

### Checklist

- [x] CI passed

### CODE REVIEW

File: docs/introduction/01-setting-up.mdx [hunk 1]
```diff
@@ -72,6 +72,11 @@ Then, run the CLI and choose from the available templates:
 pipx run crawlee create my-crawler
 ```
 
+If you already have crawlee installed . Then you can spin up crawlee by
```
Comment (B4nan) [inline docs/introduction/01-setting-up.mdx:75]:
```suggestion
If you already have `crawlee` installed, you can spin it up by running:

```

File: docs/introduction/01-setting-up.mdx [hunk 2]
```diff
@@ -72,6 +72,11 @@ Then, run the CLI and choose from the available templates:
 pipx run crawlee create my-crawler
 ```
 
+If you already have crawlee installed . Then you can spin up crawlee by 
+```sh
+crawlee create mycrawler
```
Comment (B4nan) [inline docs/introduction/01-setting-up.mdx:77]:
```suggestion
crawlee create my-crawler
```

**Review by B4nan — APPROVED** [review]

**Comment by vdusek** [comment]
@siddiqkaithodu Could you add the same thing in the readme as well please? Thanks.

**Review by vdusek — CHANGES_REQUESTED** [review]
@siddiqkaithodu Could you add the same thing in the readme as well please? Thanks.

**Comment by siddiqkaithodu** [comment]
Added @vdusek

File: README.md [hunk 3]
```diff
@@ -57,6 +57,7 @@ pip install 'crawlee[beautifulsoup,playwright]'
 
 ### With Crawlee CLI
 
+
```
Comment (B4nan) [inline README.md:60]:
```suggestion
```

**Pull request merged**